### ISSUE TITLE
The accuracy of train_data

### ISSUE THREAD

**Issue by iyouju** [deepinsight/insightface#644]
As I set softmax(logits) as pred and sent it to AccMetric,the accuracy of faces_emore that is used as train dataset was up to 0.998.But I think it should been in the range of 0.6 to 0.9.Is there any problem with this phenomenon?
